test/mongo/container.spec.ts

```diff
@@ -0,0 +1,105 @@
+import "expect-puppeteer";
+import crypto from "crypto";
+import { login } from "../utils/shared";
+
+jest.setTimeout(300000);
+
+const LOADING_STATE_DELAY = 2500;
+const RENDER_DELAY = 1000;
+
+describe("Collection Add and Delete Mongo spec", () => {
+  it("creates and deletes a collection", async () => {
+    try {
+      const dbId = `TestDatabase${crypto.randomBytes(8).toString("hex")}`;
```

Comment:
nit: would be nice to isolate this into a function like `generateUniqueName(baseName: string, length: number): string` that we reuse in other places.